src/containers/PackingOverview/RemindersPage/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const AddTodo = props => {
+  const { todoInput, handleTodoInputChange, addTodo } = props;
+
+  return (
+    <div className="btn-group dropup">
+      <button
+        type="button"
+        className="px-2 py-1 rounded-circle"
+        style={{ backgroundColor: "red" }}
```

Comment:
> style={{ backgroundColor: "red" }}

How about making a CSS style for this? It's generally not a good thing to use inline `style` attributes often, as they can make it harder for you to figure out where styles are coming from. The fewer things that set styles, the better.

You don't want to be changing things in a `.css` file then get frustrated with _"why is this background-color not updating?!"_.

Reply:
Whoops! I forgot to take that out before committing, that was just for testing purposes. Good point about the inline styling, though. 👍